docs/core/whats-new/dotnet-8.md

```diff
@@ -1335,6 +1388,18 @@ These improvements also mean that more registries are supported: Harbor, Artifac
 
 For more information containerizing .NET apps, see [Containerize a .NET app with dotnet publish](../docker/publish-as-container.md).
 
+#### Publish to tar.gz archive
+
+Starting in .NET 8, you can create a container directly as a *tar.gz* archive. This feature is useful if your workflow isn't straightforward and requires that you, for example, run a scanning tool over your images before pushing them. Once the archive is created, you can move it, scan it, or load it into a local Docker toolchain.
+
+To publish to an archive, add the `ContainerArchiveOutputPath` property to your `dotnet publish` command, for example:
+
+```dotnetcli
+dotnet publish -p PublishProfile=DefaultContainer -p ContainerArchiveOutputPath=./images/sdk-container-demo.tar.gz
```

Comment:
I want to write a new code extension that will flag when a code block is longer than 84 characters in length, as that would cause horizontal scrolling. I think this can be rewritten as follows (new-lining each property switch arg): 

```suggestion
dotnet publish \
  -p PublishProfile=DefaultContainer \
  -p ContainerArchiveOutputPath=./images/sdk-container-demo.tar.gz
```